### ISSUE TITLE
CSV input with one missing Header

### ISSUE THREAD

**Issue by HTPC-Schrauber** [jlolling/talendcomp_tFileInputTextFlat#1]
Hello,

I need to read a CSV file, which has no text in the first column of the headline. But I also need to finde the position by the name of the column.
So to say, I need a mixed mode between fixed position column and finding the column by name.

My CSV looks like this:
```
,"A","B","C","D"
"someText",10,0,8,0
"someOtherText",5,20,0,0
```

Can I achieve this somehow?

**Comment by jlolling** [comment]
Yes it is possible but not without changing the configuration of the component for each use case.

**Comment by HTPC-Schrauber** [comment]
And how? I did not get it working.
